config/settings.yml

```diff
@@ -101,6 +101,7 @@
       :omit_default_port: true
       :read_timeout: 60
   :ems_redhat:
+    :use_ovirt_engine_sdk: false
```

Comment:
The idea is that we want to get the code in and allow people who want to try it to use it but if there are problems they can turn it off. Its off by default as a safety measure.
One might want to turn this on if he wants better performance due to the improved sdk for example.
When it is tested enough the default will be true and when the old api is not supported anymore it will just be removed.